modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java

```diff
@@ -108,9 +110,10 @@ public static ObjectDefinitionGraphQLDTOContributor of(
 		List<GraphQLDTOProperty> relationshipGraphQLDTOProperties =
 			new ArrayList<>();
 
-		List<ObjectField> objectFields =
-			objectFieldLocalService.getObjectFields(
+		if (objectFields == null) {
```

Comment:
Can we replace this check for Validator.isNull ?

Reply:
SF is preventing me from doing this
![image](https://github.com/user-attachments/assets/cacf56d3-ca26-4f5e-bb26-600c76dd2879)